app/controllers/single_page_subscriptions_controller.rb

```diff
@@ -0,0 +1,58 @@
+class SinglePageSubscriptionsController < ApplicationController
+  include AccountHelper
+  include GovukPersonalisation::ControllerConcern
+
+  before_action do
+    head :not_found unless govuk_account_auth_enabled?
```

Comment:
Is this right?
[Seemed that this was enabled in staging but not on integration or prod](https://github.com/alphagov/govuk-puppet/blob/501341433129483d2bbb7ed04cb3006332dcc6de/hieradata_aws/staging.yaml#L171)?